packages/@dcl/legacy-ecs/src/decentraland/Components.ts

```diff
@@ -175,6 +176,38 @@ export class Transform extends ObservableComponent {
   }
 }
 
+/** @public */
+export enum AttachToAvatarAnchorPointId {
```

Comment:
I'd mark this `@public` as `@alpha` instead and move `AttachToAvatarAnchorPointId` to posix